Patch-1: Administrator & Educator Prompts

bencampmusic opened the pull request:
This pull request includes additional prompts for Educators and Administrators, as well as updated Readme files that link to those prompts.

bencampmusic committed:
Meeting Summary and Next Steps.MD

bencampmusic committed:
Update Meeting Summary.MD

bencampmusic committed:
Update Meeting Summary.MD

bencampmusic committed:
Meeting Summary and Agenda. CreationMD

bencampmusic committed:
Meeting Summary and Agenda Planner.MD

bencampmusic committed:
Create Individualized Student Assistance.MD

A prompt for individualized assistance with students who are struggling in class.

bencampmusic committed:
Update README.MD

Added link to individualized Student Assistance Prompt

bencampmusic committed:
Create Assignment Ideation for Active Learner 

A prompt for ideating new assignment ideas that increase student buy-in and engagement.

bencampmusic committed:
Rename Assignment Ideation for Active Learner to Assignment Ideation for Active Learner.MD

Adding file extension

bencampmusic committed:
Update README.MD

Add link to Active Learner Assignments

bencampmusic committed:
Update README.MD

Added link to Meeting Summary prompt

bencampmusic committed:
Merge pull request #1 from bencampmusic/main

Moving Main to Patch-1

cricex commented:
Thanks for the submission! We are reviewing these prompts.

cricex commented on Administration/Prompts/Meeting Summary.MD:
```diff
@@ -0,0 +1,105 @@
+this summary. Do not repeat yourself, avoid redundant language. Be clear and concise.
+
+Your administrator will provide you with meeting notes or a meeting transcript. After 
+each input from the administrator ask "Is this the full notes or is there more?"  You 
```
@bencampmusic 
It seems to immediately ask whether the full notes have been submitted even before any notes have been submitted. 

I think we should be more explicit to only ask if all notes have been submitted after each note submission instead of "After each input."
 
Example interaction (Bing Chat Enterprise): 
![image](https://github.com/microsoft/prompts-for-edu/assets/7026048/0b6bcc37-5302-4e25-852f-5107b1ae5685)

bencampmusic replied:
Thanks for catching this!

I've submitted a new pull request with an updated prompt that should address this issue.  Let me know if you have any other thoughts or suggested changes.

-Ben

bencampmusic committed:
Update Meeting Summary.MD with edits for @cricex

Updated prompt instructions to avoid LLM responding inappropriately or prematurely.
